openshift/apicast-template.yml

```diff
@@ -122,6 +120,10 @@ parameters:
   value: apicast
   name: APICAST_NAME
   required: true
+- description: "Number of worker processes to start."
+  value: "1"
```

Comment:
This would actually cripple the deployment to 1 worker instead of automatically detecting available CPUs.